src/domain/safe/entities/__tests__/module-transaction.factory.ts

```diff
@@ -9,27 +8,27 @@ export default function (
   created?: string,
   data?: string | null,
   dataDecoded?: DataDecoded | null,
-  executionDate?: Date,
+  executionDate?: string,
```

Comment:
I changed the Factory because is testing purpose and the ModuleTransaction validator waits for a string in a date instead a Date object.

Reply:
Ok, I'll modify the entity to return it from the factory, for other domains it's preferable do it in other PR.

Reply:
Done https://github.com/5afe/safe-client-gateway-nest/pull/205/commits/1905fe70073404b8b1129454be61eed7ff3803da